File: connectors/connector-mantis/src/test/scala/com/taskadapter/connector/mantis/MantisFieldBuilder.scala
package com.taskadapter.connector.mantis

import com.taskadapter.connector.FieldRow
import com.taskadapter.model.{Description, Summary}

object MantisFieldBuilder {
  def getDefault(): List[FieldRow[_]] = {
    List(
      FieldRow.apply(Summary, Summary, ""),
      FieldRow.apply(Description, Description, "-")
    )
  }
}

File: connectors/connector-mantis/src/test/java/com/taskadapter/connector/mantis/MantisFieldBuilder.java
package com.taskadapter.connector.mantis;

import com.taskadapter.connector.FieldRow;
import com.taskadapter.model.AllFields;

import java.util.List;

public class MantisFieldBuilder {
    public static List<FieldRow<?>> getDefault() {
        return List.of(
                FieldRow.apply(AllFields.summary, AllFields.summary, ""),
                FieldRow.apply(AllFields.description, AllFields.description, "-")
        );
    }
}
